### PR TITLE
IAM-460-Update login-ui relation

### PR DESCRIPTION
**Pull request by bencekov**
As part of IAM-460, I took out the unused pages from the login-ui backend. For this reason, the login-ui-relation relation was also updated to not include invalid links.

### CODE REVIEW

File: lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py [hunk 1]
```diff
@@ -61,10 +61,8 @@ def some_event_function():
 RELATION_KEYS = [
     "consent_url",
     "error_url",
-    "index_url",
     "login_url",
     "oidc_error_url",
-    "registration_url",
     "default_url",
```
Comment (nsklikas) [inline lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py:66]: If you remove the index page, then you should update the `default_url` as well (it used to be the `index` page)

File: lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py [hunk 2]
```diff
@@ -111,10 +109,8 @@ def send_endpoints_relation_data(self, endpoint: str) -> None:
             endpoint_databag = {
                 "consent_url": f"{endpoint}/ui/consent",
                 "error_url": f"{endpoint}/ui/error",
-                "index_url": f"{endpoint}/ui/index",
                 "login_url": f"{endpoint}/ui/login",
                 "oidc_error_url": f"{endpoint}/ui/oidc_error",
-                "registration_url": f"{endpoint}/ui/registration",
                 "default_url": endpoint,
```
Comment (nsklikas) [inline lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py]: If you remove the index page, then you should update the `default_url` as well (it used to be the `index` page)

Comment (bencekov) [reply lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py]: I could do that, but I already implemented the kratos operator to use the login page as the default return page. Do you still need this change?

Comment (nsklikas) [reply lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py]: Then what's the point of having a `default_url` in the databag? Let's remove it.

Comment (bencekov) [reply lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py]: ok

**Review by nsklikas — APPROVED** [review]

**Pull request merged**